services/agent/tests/unit/test_services_backup.py

```diff
@@ -37,6 +40,30 @@ def volume_content(tmpdir: Path) -> Path:
     return path
 
 
+@pytest.fixture
+async def mock_container_with_data(
+    volume_content: Path, monkeypatch: pytest.MonkeyPatch
+) -> AsyncIterable[None]:
+    async with aiodocker.Docker() as client:
+        container = await client.containers.run(
+            config={
+                "Image": "alpine:latest",
+                "Cmd": ["/bin/ash", "-c", "sleep 10000"],
+                "HostConfig": {"Binds": [f"{volume_content}:{volume_content}:rw"]},
+            }
+        )
+        container_inspect = await container.show()
+
+        container_name = container_inspect["Name"][1:]
+        monkeypatch.setenv("HOSTNAME", container_name)
+
+        yield None
+
+        with contextlib.suppress(aiodocker.DockerError):
+            await container.kill()
+        await container.delete()
+
```

Comment:
why the kill? if you need `remove -f` I think there is such an argument already in delete...
and technically it is not a mocked container, but a real container with data.